app/src/main/res/layout/fragment_page.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I think we can get rid of the `LinearLayout` and use `ConstraintLayout` directly. This way we'll get rid of nesting 2 layouts.